.github/workflows/publish.yml

```diff
@@ -18,5 +18,8 @@ jobs:
         with:
           python-version: '3.12'
       - uses: snok/install-poetry@v1
+        with:
+          plugins: |
+            poetry-dynamic-versioning[plugin]
```

Comment:
Any reference for this change? Isn’t it fixing the second commit?
Please put both answers in the third commit message.

Reply:
Merged with the previous commit.